articles/concepts-multiple-qubits.md

```diff
@@ -95,15 +99,15 @@ $$
 > [!NOTE]
 > This article uses the *little-endian* format to label the computational basis. In little endian format, the least significant bits come first. For example, the number four in little-endian format is represented by the string of bits 001.
 
-Since each qubit measurement can only yield $0$ or $1$, the probability of measuring $0$ is simply $1-P(\text{outcome}=1)$.  This is why you only explicitly give a formula for the probability of measuring $1$.
+Since each qubit measurement can only yield $0$ or $1$, the probability of measuring $0$ is simply $1-P(\text{outcome}=1)$.  This is why you only need a formula for the probability of measuring $1$.
 
 The action that such a measurement has on the state can be expressed mathematically as
 
 $$
 \psi \mapsto \frac{\sum_{e_k \text{ in the set } S} e_k e_k^\dagger \psi}{\sqrt{P(\text{outcome}=1)}}.
 $$
 
-The cautious reader may worry about what happens when the probability of the measurement is zero.  While the resultant state is technically undefined in this case, you never need to worry about such eventualities because the probability is zero!
+The cautious reader may worry about what happens if the denominator is zero.  While such state is undefined, it cannot occur because the probability is zero!
```

Comment:
```suggestion
The cautious reader may worry about what happens if the denominator is zero.  While such state is undefined, you don't need to worry about such eventualities because the probability is zero! 
```